resolwe/flow/models/utils/__init__.py

```diff
@@ -0,0 +1,83 @@
+"""Resolwe models utils."""
+from resolwe.flow.utils import iterate_fields
+
+from .hydrate import (  # noqa: F401
+    hydrate_input_references,
+    hydrate_input_uploads,
+    hydrate_size,
+)
+from .reference import referenced_files  # noqa: F401
+from .render import (  # noqa: F401
+    fill_with_defaults,
+    json_path_components,
+    render_descriptor,
+    render_template,
+)
+from .validation import (  # noqa: F401
+    DirtyError,
+    validate_data_object,
+    validate_process_types,
+    validate_schema,
+    validation_schema,
+)
+
+
+def get_collection_of_input_entities(data):
+    """Get collection that contains all "entity inputs" of a given data.
+
+    With "entity input", one refers to the inputs that are part of an entity.
```

Comment:
I would add a paragraph about ValueError that is raise when inputs are part of more than one collection.